.changelog/release-v14.0.0.md

```diff
@@ -70,9 +103,19 @@
 
 ## 5. Breaking changes
```

Comment:
```suggestion
## 5. Breaking changes
```

The changes below are breaking for upgrades and implementations of a Mojaloop Switch but not for FSPs or other entities interacting with a Mojaloop Switch itself. The API changes are non-breaking.

Reply:
I believe the API changes are breaking. Some of the responses have changed elements such as the party response with `id` --> `address` (e.g. https://github.com/mojaloop/testing-toolkit-test-cases/pull/76/files#diff-ce30aada2c33a906ee2d0bad0b193cc7650e1d70b21b6ca348428b16a731344fL74(